plugins/formal-verification/formver.viper/src/org/jetbrains/kotlin/formver/viper/ast/Exp.kt

```diff
@@ -426,5 +426,5 @@ sealed interface Exp : IntoSilver<viper.silver.ast.Exp> {
         info: Info = Info.NoInfo,
         trafos: Trafos = Trafos.NoTrafos,
     ): AccessPredicate.FieldAccessPredicate =
-        AccessPredicate.FieldAccessPredicate(fieldAccess(field), permission, pos, info, trafos)
+        AccessPredicate.FieldAccessPredicate(fieldAccess(field, pos), permission, pos, info, trafos)
```

Comment:
Is this position information correct?  Both the field access and the access predicate have the same position information?